src/lib/sources/nettruyen.py

```
@@ -0,0 +1,27 @@
+import requests
+from bs4 import BeautifulSoup
+from .base import ComicSource
+from typing import Any, Dict
+
+class NettruyenSource(ComicSource):
+    def fetch_page(self, url: str) -> str:
+        resp = requests.get(url)
```

Ruff (0.13.1)

Probable use of `requests` call without timeout

(S113)